sdk/devcenter/Azure.Developer.DevCenter/tests/DevCenterClientTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Developer.DevCenter.Tests
+{
+    public class DevCenterClientTests : RecordedTestBase<DevCenterClientTestEnvironment>
+    {
+        private DevCenterClient _devCenterClient;
+
+        internal DevCenterClient GetDevCenterClient() =>
+            InstrumentClient(new DevCenterClient(
+                TestEnvironment.Endpoint,
+                TestEnvironment.Credential,
+                InstrumentClientOptions(new DevCenterClientOptions())));
+
+        public DevCenterClientTests(bool isAsync) : base(isAsync)
```

Comment:
How are these tests running w/o the azure resource creation? Am I missing something?

Reply:
Azure resources get created using [test-resources.bicep](https://github.com/Azure/azure-sdk-for-net/blob/3f1978a4c53fae22f73f6496337b7530df8e9abb/sdk/devcenter/test-resources.bicep) file and [here](https://github.com/Azure/azure-sdk-for-net/blob/3f1978a4c53fae22f73f6496337b7530df8e9abb/sdk/devcenter/Azure.Developer.DevCenter/tests/DevCenterClientTestEnvironment.cs#L18-L28) we get it from the env variables.

Reply:
In the other test file they had mentioned they are not creating the resource since it can take a long time and they rather want to run everything in playback mode.

Reply:
Right, we generate the necessary resources with the bicep script, run live tests to make the records, then run tests against the records.

Create/delete/start/stop operations on Dev Boxes and Environments are LROs. Running the entire test suite might take hours (~3-4 hours in my experience).